### PR TITLE
Add Pins for Amazon CAs

### PR DESCRIPTION
**Pull request by jumde**
Fixes: https://github.com/brave/muon/issues/526

### CODE REVIEW

**Comment by bridiver** [comment]
@jumde can you add label(s)/milestone?

**Comment by bridiver** [comment]
is this change specifically for C65?

**Comment by jumde** [comment]
This is required for updating the certs for services like sync.brave.com. I think we need this for C65 cc: @ayumi @diracdeltas

**Comment by bridiver** [comment]
I'm asking because 4.9 is for C65 specific changes. If this isn't necessary for C65 it should be moved to 5.x

**Comment by diracdeltas** [comment]
we can wait until after c65 for this. it is needed so that various brave services can switch to using AWS's cert manager.

**Comment by diracdeltas** [comment]
lgtm. it would be good to set up a test pinning domain (like pinning-test.brave.com) on AWS with an AWS cert to test this.

File: patches/master_patch.patch [hunk 1]
```diff
@@ -1870,7 +1872,7 @@ index 023555bcd3b56fd12c3319096deff07fa9854388..5b75bfaa858cead732a701029f45f549
  
    bool ShouldSelectReplacement() const {
 diff --git a/third_party/WebKit/Source/core/exported/WebViewImpl.cpp b/third_party/WebKit/Source/core/exported/WebViewImpl.cpp
-index b73e62a0420f77db2874260a6f5e141264018cb8..c0f24ad844cb53eb813e4dc6ad7f8eef20ca7fc4 100644
+index 9a59d4f569de1ea8f5cd84849e313b4512576043..8e12398d8a153f9710650e35f21b1f5b9de83dd3 100644
```
Comment (bridiver) [inline patches/master_patch.patch]: this looks unrelated

**Comment by bsclifton** [comment]
reverted from master with https://github.com/brave/muon/commit/0b57d3e3a357871b2d2345d0e5a91b75d1fdb111

reverted from `5.1.x+65.0.3325.181` with https://github.com/brave/muon/commit/6b284144bd643f437d14ea565724534c049fd381
